qrcode/image/svg.py

```diff
@@ -1,6 +1,4 @@
 from decimal import Decimal
-# On Python 2.6 must install lxml since the older xml.etree.ElementTree
-# version can not be used to create SVG images.
```

Comment:
Maybe change the comment to mention why `lxml` is stil preferred.

Reply:
Sure, why is it still preferred? Does it have better performance?

Reply:
I have no idea. :D

But it probably doesn’t matter, bench it if you care but otherwise you can probably ignore my comment. AFAIK `ElementTree` got rewritten in C at some point between python2 and python3.